dev/com.ibm.ws.crypto.passwordutil/src/com/ibm/ws/crypto/util/AESKeyManager.java

```diff
@@ -67,8 +70,14 @@ private KeyHolder get(char[] keyChars) {
                     // Still use this holder for returns even if I do not end up caching it.
                     holder = holder2;
                 } catch (InvalidKeySpecException e) {
+		    if (Boolean.getBoolean("enableDebug")) {
+			securityUtilDebug("InvalidKeySpecException received. Returning null", e); 
+		    }
                     return null;
                 } catch (NoSuchAlgorithmException e) {
+		    if (Boolean.getBoolean("enableDebug")) {
```

Comment:
Shouldn't we use `DEBUG` here, instead of making the getBoolean call again?

Reply:
Same as above, fixed.